tools/common_lib/src/gemm.h

```diff
@@ -944,15 +948,15 @@ class GemmUmdD3d12Dispatcher : public GemmBaseDispatcher
     {
         using namespace dnnl_utils;
 
-        input_a_memory_desc_ = to_dnnl_mem_desc(params_.shape_a, params_.layout, params_.dt);
-        const auto input_b_memory_desc = to_dnnl_mem_desc(params_.shape_b, params_.b_managed ? DataLayout::eWeightsLayoutStart : params_.layout, params_.dt);
+        input_a_memory_desc_ = to_dnnl_mem_desc(params_.a_transposed ? TensorShape{ params_.shape_a.n, params_.shape_a.c, params_.shape_a.w, params_.shape_a.h } : params_.shape_a, params_.layout, params_.dt);
```

Comment:
I wonder if the oneDNN shouldnt work differently:
1. Always set the tensor shape as normal 
2. Transpose is done by setting different dnnl::memory::format_tag (i.e. nchw -> ncwh)
Can you check and comment?

Reply:
I had thought about using that but when I referenced the documentation, I didn't find anything for ncwh.
This is the link I had referenced: https://oneapi-src.github.io/oneDNN/enum_dnnl_memory_format_tag.html